ci/restore_questionnaire_state.yaml

```diff
@@ -17,6 +17,17 @@ run:
   args:
     - -exc
     - |
+      VARS=('PROJECT_ID' 'BUCKET_NAME' 'BACKUP_NAME' 'FILE_PREFIX')
```

Comment:
Really minor but i I'd call these `PARAMS` rather than vars to keep it consistent throughout the file.

Reply:
Added, thanks.